src/CloudFlare/Zone/Settings.php

```diff
@@ -471,6 +471,8 @@ public function change_ipv6($zone_identifier, $value = 'off') {
 	 * @return array|mixed
 	 */
 	public function change_minify($zone_identifier, $value) {
+		
+		$data = ['value' => $value];
```

Comment:
Oops. That's what you get when you try to make a PR straight from the GH interface!